docs/releases/3.2.txt

```diff
@@ -606,3 +609,8 @@ Miscellaneous
 * The ``default_app_config`` application configuration variable is deprecated,
   due to the now automatic ``AppConfig`` discovery. See :ref:`whats-new-3.2`
   for more details.
+
+* Calling ``repr()`` on a queryset in
```

Comment:
Automatically calling...

Reply:
Or maybe we should refer to the fact that the default value of the `transform` argument is changing? Users may not be aware that ``repr()`` is being called for them anyway unless they've played with that argument?

Reply:
It's under the "Features deprecated in 3.2" so an immediate change in the default values doesn't fit here, IMO.